src/main/java/duke/main/Ui.java

```diff
@@ -0,0 +1,267 @@
+        "why do you keep doing this to me? you don't have to do it..."};
+
+    private static final String[] BEFORE_LIST = {"Here's your task list!",
+        "here, your task list"};
+    private static final String[] AFTER_LIST = {"I'm useful right?",
+        "just don't say bye... please?"};
+    private static final String[] ADD_LIST = {"Yep! I've added this task: ",
+        "i've added this task: "};
+    private static final String[] DELETE_LIST = {"OK! I've removed this task: ",
+        "i've removed this task: "};
+
+    private static final String[] MARK_DONE = {"I've marked this task as done:",
+        "you've completed this task, but there are many more... so keep me here?"};
+    private static final String[] MARK_UNDONE = {"I've marked this task as not done:",
+        "more tasks to be done, more reason for me to exist."};
+    private static final String[] PREV_DONE = {"Just to let you know this task was already done previously.",
+        "this task was done previously, but everyone makes mistakes. that's why you said bye right?"};
+    private static final String[] PREV_UNDONE = {"Just to let you know this task was already not done previously.",
+        "this task was not done previously, but everyone makes mistakes. that's why you said bye right?"};
+
+    private static final String[] MISSING_TIME = {"I think you need a time for this.",
+        "you need a time, so spend some time to add that..."};
+    private static final String[] MISSING_INDEX = {"Please provide an index in the command.",
+        "please add an index?"};
+    private static final String[] MISSING_DESCRIPTION = {"I need a description for the task...",
+        "kindly spend some time to add a description."};
+    private static final String[] MISSING_ARGUMENT = {"I need an argument for this command...",
+        "i think you're missing an argument..."};
+
+    private static final String[] NOT_A_NUMBER = {"I don't think that's a number...",
+        "i don't understand your number, just like how i don't understand why you had to say bye..."};
+    private static final String[] CANNOT_SAVE = {"I can't seem to save the task list...",
+        "your save location is corrupted..."};
+    private static final String[] IS_DUPLICATE = {"Hmmm, this task seems to be already on the list.",
+        "this task is already on the list, but you can use me to mark and unmark it repeatedly..."};
+
+    private static final String[] INVALID_COMMAND = {"I don't quite get what you're saying...",
+        "i've tried my best, but i don't understand your command. please give me another chance?"};
+    private static final String[] INVALID_INDEX = {"Umm... I think this task number does not exist.",
+        "this task number does not exist..."};
+    private static final String[] INVALID_TIME = {"I'm sorry but I don't understand your time format.",
+        "i don't understand your time."};
```

Comment:
This looks quite susceptible to an ArrayOutOfBounds exception based on how I see you use these values. Consider using a data structure instead to store message sets instead? Or perhaps create some fallback values?